chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,709 +1,806 @@
+									"    }\r",
+									"  ],\r",
+									"  \"transparencyAttestations\": [\r",
+									"    {\r",
+									"      \"acbId\": 0,\r",
+									"      \"acbName\": \"string\",\r",
+									"      \"attestation\": {\r",
+									"        \"removed\": false,\r",
+									"        \"transparencyAttestation\": \"string\"\r",
+									"      }\r",
+									"    }\r",
+									"  ],\r",
+									"  \"website\": \"string\"\r",
+									"};\r",
+									"pm.test(\"Validate that /hierarchy end point has correct schema\", () => {\r",
+									"            pm.response.to.have.jsonSchema(schema);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "API-Key",
+								"value": "{{apiKey}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/rest/developers/{{developerId}}/hierarchy",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"developers",
+								"{{developerId}}",
+								"hierarchy"
+							]
+						},
+						"description": "/hierarchy end point returns 200 status and correct response for valid Developer Ids"
+					},
+					"response": []
+				},
+				{
+					"name": "GET /hierarchy - Invalid developer Id",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "f5f06c65-6b79-4c85-8a5d-3a73ef785980",
+								"exec": [
+									"pm.test(\"Invalid DeveloperId returns 404 status\", function () {\r",
+									"    pm.response.to.have.status(404);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "API-Key",
+								"value": "{{apiKey}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/rest/developers/100/hierarchy",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"developers",
+								"100",
+								"hierarchy"
+							]
+						},
+						"description": "Invalid Developer Id returns 404 status"
+					},
+					"response": []
+				},
+				{
+					"name": "GET /developers/deverloperId ",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "1f95c359-f52d-4b60-abd8-5ab5fa72b7dd",
+								"exec": [
+									"const developerId= pm.iterationData.get(\"developerId\");\r",
+									"pm.environment.set(\"developerId\",\"developerId\");"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "8648433c-dd18-47d0-ad72-e3549538a36f",
```

Comment:
Need to remove Ids

Reply:
this has been there when those tests were added. I guess I can remove it

Reply:
i removed a few unused ids from the file

Reply:
pushed a commit with that update